lib/Installation/Partitioner/LibstorageNG/FileSystemOptionsPage.pm

```diff
@@ -28,4 +29,11 @@ sub press_next {
     $self->get_navigation_panel()->press_next();
 }
 
+sub set_separate_home_partition_checkbox {
```

Comment:
aren't we repeating this method here and in FileSystemOptionsLvmPage and the only difference is the shortcut to initialize the object.

Reply:
So, File System Options screen currently has different shortcuts and checkbox names depending on whether lvm was selected on the previous screen or not. 

I wanted to make two implementations for the page to not mix up the logic in one object. And the methods are called as the checkboxes on the pages: 'set_separate_home_**partition**_checkbox' and 'set_separate_home_**volume**_checkbox'.

The other solution, that I see is to make kind of 'protected' method '_set_separate_home_by_shortcut($shortcut)' and call it in both places with the different shortcut parameter.

What do you think? Do you see some other and better solution here? Could you please share your thoughts.

Reply:
Ahm, I see now, thanks (I didn't see that difference in your VR, but in prod is clear). It smells to interfaces, but no idea in perl. I like more like is now than with something with `shortcut` in the name of the function. I will think about it. Still trying to connect pieces of this approach.